scifight_proj/settings.py

```diff
@@ -110,7 +110,16 @@
          label='Tournament progress',
          models=[
              'scifight.Fight',
-             'scifight.FightStage']),
+             'scifight.FightStage',
+             'scifight.Jury',
+             dict(model='scifight.JuryPoints', label='Marks of jury'),
+             ]),
+
+    dict(app='scifight',
+         label='Origin',
+         models=[
+             'scifight.TeamOrigin',
+             'scifight.CommonOrigin']),
 
```

Comment:
It seems like you have a merge artifact here, remove this diff hunk from the commit.